sql/information-schema.md

```diff
@@ -0,0 +1,441 @@
++------------------------+--------------------+
+| COLLATION_NAME         | CHARACTER_SET_NAME |
++------------------------+--------------------+
+| utf8mb4_general_ci     | utf8mb4            |
+| utf8mb4_bin            | utf8mb4            |
+| utf8mb4_unicode_ci     | utf8mb4            |
+| utf8mb4_icelandic_ci   | utf8mb4            |
+| utf8mb4_latvian_ci     | utf8mb4            |
+| utf8mb4_romanian_ci    | utf8mb4            |
+| utf8mb4_slovenian_ci   | utf8mb4            |
+| utf8mb4_polish_ci      | utf8mb4            |
+| utf8mb4_estonian_ci    | utf8mb4            |
+| utf8mb4_spanish_ci     | utf8mb4            |
+| utf8mb4_swedish_ci     | utf8mb4            |
+| utf8mb4_turkish_ci     | utf8mb4            |
+| utf8mb4_czech_ci       | utf8mb4            |
+| utf8mb4_danish_ci      | utf8mb4            |
+| utf8mb4_lithuanian_ci  | utf8mb4            |
+| utf8mb4_slovak_ci      | utf8mb4            |
+| utf8mb4_spanish2_ci    | utf8mb4            |
+| utf8mb4_roman_ci       | utf8mb4            |
+| utf8mb4_persian_ci     | utf8mb4            |
+| utf8mb4_esperanto_ci   | utf8mb4            |
+| utf8mb4_hungarian_ci   | utf8mb4            |
+| utf8mb4_sinhala_ci     | utf8mb4            |
+| utf8mb4_german2_ci     | utf8mb4            |
+| utf8mb4_croatian_ci    | utf8mb4            |
+| utf8mb4_unicode_520_ci | utf8mb4            |
+| utf8mb4_vietnamese_ci  | utf8mb4            |
++------------------------+--------------------+
+26 rows in set (0.00 sec)
+```
+
+## COLUMNS Table
+
+COLUMNS 表提供了关于所有表的列的信息。
+
+```sql
+mysql> CREATE TABLE test.t1 (a int);
+1 row in set (0.01 sec)
+mysql> SELECT * FROM information_schema.columns WHERE table_schema='test' AND TABLE_NAME='t1'\G
+*************************** 1. row ***************************
+           TABLE_CATALOG: def
+            TABLE_SCHEMA: test
+              TABLE_NAME: t1
+             COLUMN_NAME: a
+        ORDINAL_POSITION: 1
+          COLUMN_DEFAULT: NULL
+             IS_NULLABLE: YES
+               DATA_TYPE: int
+CHARACTER_MAXIMUM_LENGTH: NULL
+  CHARACTER_OCTET_LENGTH: NULL
+       NUMERIC_PRECISION: 11
+           NUMERIC_SCALE: 0
+      DATETIME_PRECISION: NULL
+      CHARACTER_SET_NAME: NULL
+          COLLATION_NAME: NULL
+             COLUMN_TYPE: int(11)
+              COLUMN_KEY: 
+                   EXTRA: 
+              PRIVILEGES: select,insert,update,references
+          COLUMN_COMMENT: 
+   GENERATION_EXPRESSION: 
+1 row in set (0.01 sec)
+```
+
+The corresponding `SHOW` statement is as follows:
+
+```sql
+mysql> SHOW COLUMNS FROM t1 FROM test;
++-------+---------+------+------+---------+-------+
+| Field | Type    | Null | Key  | Default | Extra |
++-------+---------+------+------+---------+-------+
+| a     | int(11) | YES  |      | NULL    |       |
++-------+---------+------+------+---------+-------+
+1 row in set (0.00 sec)
+```
+
+## ENGINES Table
+
+`ENGINES` 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB 一直将 InnoDB 描述为唯一支持的引擎。
```

Comment:
```suggestion
`ENGINES` 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB 会一直将 InnoDB 描述为唯一支持的引擎。
```

Reply:
Please ignore the previous comment and see the one below (the `Delete` button does not work currently):